src/components/pages/accounts/IncomeExpenseHistogram.tsx

```diff
@@ -52,17 +52,6 @@ export default function IncomeExpenseHistogram({
     },
   ];
 
-  dates.forEach(date => {
-    const monthYear = (date as DateTime).toFormat('MM/yyyy');
```

Comment:
the data format has changed, if we request monthly totals for 6 months then we return an array of 6 elements with `AccountsTotals` type.